planpro_importer/planpro110/signalreader.py

```diff
@@ -0,0 +1,116 @@
+import logging
+
+from yaramo.model import Signal, SignalState, SignalFunction, SignalKind, SignalSystem
+from typing import Set
+
+
+class SignalReader:
+
+    def __init__(self, topology, container):
+        self.topology = topology
+        self.container = container
+
+    def get_signal_frames_by_signal_uuid(self, signal_uuid: str):
+        frames = []
+        for frame in self.container.Signal_Rahmen:
+            if frame.ID_Signal.Wert == signal_uuid:
+                frames.append(frame)
+        return frames
+
+    def get_signal_terms_by_signal_frame_uuid(self, signal_frame_uuid: str) -> Set[SignalState]:
```

Comment:
Generally, more docstrings would help reading the code. At the same time, I see that we do this very inconsistently anyway.